cli_tools/diagnostics/main_windows.go

```diff
@@ -256,6 +256,14 @@ func getPlainEventLogs(evts []winEvt, errs chan error) []string {
 	return runAll(commands, errs)
 }
 
+// getDockerImagesList put docker images list file in logFolder channel and errors in error channel.
+func getDockerImagesList(logs chan logFolder, errs chan error) {
+	var commands = []runner{
+		cmd{`C:\Program Files\Docker\docker.exe`, "image list", "docker_images.log", false},
```

Comment:
Remind me, what happens if we invoke this and docker isn't installed on the VM? Does this lead to any failures or error output for the user, or do we just not do anything?

Reply:
It won't break anything, we will only log a general error message in serial port

Reply:
Would it be possible to suppress that error message? (same for other already-existing checks for docker / kubernetes logs). If there's a reasonable way to do this then that would be nice, but not a deal-breaker.

Reply:
Yeah, we suppress it like other logs and print one general message like `Errors occured while collecting and zipping some logs...` in serial port.